README.md

```diff
@@ -50,162 +43,112 @@ source and public models in popular formats such as TensorFlow, ONNX, PaddlePadd
-    </tbody>
-</table>
-
-OpenVINO™ Toolkit also contains several plugins which simplify loading models on several hardware devices:
-<table>
-    <thead>
-        <tr>
-            <th>Plugin</th>
-            <th>Library</th>
-            <th>Short Description</th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/auto-device-selection.html">Auto</a></td>
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Auto plugin enables selecting Intel device for inference automatically</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/automatic-batching.html">Auto Batch</a></td>
-            <td><b><i><a href="./src/plugins/auto_batch">openvino_auto_batch_plugin</a></i></b></td>
-            <td>Auto batch plugin performs on-the-fly automatic batching (i.e. grouping inference requests together) to improve device utilization, with no programming effort from the user</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/hetero-execution.html">Hetero</a></td>
-            <td><b><i><a href="./src/plugins/hetero">openvino_hetero_plugin</a></i></b></td>
-            <td>Heterogeneous execution enables automatic inference splitting between several devices</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/multi-device.html">Multi</a></td>
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Multi plugin enables simultaneous inference of the same model on several devices in parallel</td>
-        </tr>
-    </tbody>
-</table>
-
-## License
-OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
-By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
 
-## Telemetry
-OpenVINO™ collects software performance and usage data for the purpose of improving OpenVINO™ tools. This data is collected directly by OpenVINO™ or through the use of Google Analytics 4.
-You can opt-out at any time by running the command:
+### Getting Started
 
-``` bash
-opt_in_out --opt_out
-```
+[OpenVINO Quickstart example](https://docs.openvino.ai/2024/notebooks/201-vision-monodepth-with-output.html) will walk you through the basics to deploy your first model.
+
+### Installation
 
-More Information is available at https://docs.openvino.ai/latest/openvino_docs_telemetry_information.html.
+Get your OpenVINO installation command with just a few clicks. [Go to the installation page](https://docs.openvino.ai/2024/openvino_docs_install_guides_overview.html).
```

Comment:
Let us move `Installation` to the very top. And no need to have a link to instruction. It wastes more space than if we explicitly share instruction how to install:

```
pip install openvino
```

Reply:
Agree, documentation should reflect natural flow of software usage - first you have to install, then - start using